src/DataTable/README.md

```diff
@@ -275,20 +275,27 @@ To enable proper selection behavior with backend pagination (i.e., when ``isSele
         },
       ]}
       bulkActions={[
-        {
-          buttonText: 'Download CSV',
-          handleClick: (data) => console.log('Download CSV', data),
-        },
-        // custom button function that utilizes clearSelection function provided by the table instance
-        ({tableInstance})=>{
-          return {
-            buttonText: `Clear Selection`,
-            handleClick: () => {
-              console.log('Clear selection');
-              tableInstance.clearSelection()
+        ({ as, selectedFlatRows }) => React.createElement(
+          as || Button,
+          {
+            key: 'Download CSV',
+            onClick: () => console.log('Download CSV', selectedFlatRows),
+            className: as ? '' : 'ml-2',
```

Comment:
It'd also be great if the user didn't need to consider the spacing between buttons, but rather had `DataTable` take care of that itself through a SCSS property, e.g.:

```scss
.pgn__bulk-actions,
.pgn__table-actions {
  display: flex;
  margin-left: map_get($spacers, 2);

  // new
  .btn {
    margin-left: map_get($spacers, 1);
    :first-child {
      margin-left: 0;
    }
  }
}
```

Reply:
Yes, I added this snippet to the code